ci/dependency-update/rules-orm6.5plus.xml

```diff
@@ -0,0 +1,12 @@
+<ruleset comparisonMethod="maven"
+         xmlns="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0 http://mojo.codehaus.org/versions-maven-plugin/xsd/rule-2.0.0.xsd">
+  <ignoreVersions>
+    <!-- Ignore Alpha/Beta/CRs versions in the repo, because the versions-maven-plugin considers those
+         as higher priority than snapshots, which is wrong. -->
+    <ignoreVersion type="regex">.*\.(Alpha|Beta|CR).*</ignoreVersion>
+    <!-- Restrict allowed versions to a particular major/minor, because forbidding major/minor upgrades
+         in the version-maven-plugin configuration doesn't always work for some reason. -->
+    <ignoreVersion type="regex">(?!6\.[5-9]\.).*</ignoreVersion>
```

Comment:
I don't think we need 6.5+, just 6.5.0+. People wanting to use 6.6 (not even planned yet) will most likely be required to use Search 7.2.